### DISCUSSION TITLE
Workflow with Readarr

### DISCUSSION THREAD

**Discussion by CedricPelchat** [Neurrone/beets-audible#53] [Q&A]
Hello! Thanks for this great plugin!
I am using Readarr to download audiobooks which get put in  a file structure like this: {Artists name}/{book name} - {book part}.
For now, I'm outputting those files to the recentlyadded folder of the auto m4b tool so that it joins mp3 files together. The thing is, that tool uses only the folder name as title for the book, so books gets renamed by their authors name. Auto-m4b recommends using beets after merging files. If i overwrite the book title from the filename, will I be able to use beets correctly? Should I try to rename files with beets before using auto-m4b? I dont see any workaround for getting an author folder with readarr.

**Comment by Neurrone** [comment]
I don't have any experience with Readarr, so I'd say try it and see what results you get with the plugin.

**Reply by CedricPelchat** [reply]
Does your plugin get the title of the books from the name of the file? If so, I'll have a problem, since my files get renamed as the author's name.

**Reply by CedricPelchat** [reply]
I'm sorry, that was answered in the documentation. One last question: In auto-m4b's documentation, it says that it is meant to be used with this plugin. It put files in a 'untagged' folder, which I guess I'll put as path/to/import/from in beets. However, when there is still some merging going on, there are some temporary files in there. Book files get put in that structure temporarily: book/booktempfiles/bookconverting.m4b. Since the files are nested in a folder, does that prevent beets plugin from picking them up?

**Reply by Neurrone** [reply]
No, it should still try to process them, although I'm not 100% sure since I've not tried on a folder structure like what you've described.
